.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
-  - "0.10"
-  - "0.12"
```

Comment:
alright, I did some research and can't find any evidence that these are system defaults so they're probably safe to remove.